src/main/resources/test_files/runtime/Operational/AdditiveOp2.iq

```diff
@@ -0,0 +1,10 @@
+(:JIQS: ShouldRun; Output="(2, 2.0, 20.0, 2.0, 11.0, 11.0)" :)
+1 + 1,
```

Comment:
Are there any tests for other operations (*, -, div...)?

Reply:
Some of these tests existed in various files. These have been reorganized in the following commits. And more are introduced in this PR and other PR's which originate from this branch.